### ISSUE TITLE
Default queue gets distributed to all queues in Redis

### ISSUE THREAD

**Issue by m1racoli** [celery/celery#2713]
Hi,
we are currently running Airflow 1.1.1 (with Celery as the task execution framework) and observed a weird behaviour regarding the default-queue setting for Celery and don't know if it's intended behaviour or a bug.

If an operators queue happens to be set to the same queue as the default-queue set up in the Celery configuration, then the task gets submitted by the scheduler to each queue currently registered in Redis.

We expected that the default queue is only crucial for the worker to listen to, when it is not assigned to a specific queue.

**Comment by thedrow** [comment]
Can you provide a test case I can work with to reproduce the problem?

**Comment by m1racoli** [comment]
I tried to reproduce the behaviour on a separate setup on my local machine, but I couldn't. Still it occurred on our server.

I suspected malformed data in Redis. That's why we flushed all Celery data in Redis. After that I could not reproduce the behaviour on our server as well.

It seemed like that at the end temporarily async default_queue configurations among the workers might have caused the problem, which was the case for some time. Only bringing the config in sync didn't help, but flushing all the data in Redis did.

Best
